.eslintrc.js

```diff
@@ -6,6 +6,8 @@ module.exports = {
     'import/no-unresolved': 'error',
     // Since React 17 and typescript 4.1 you can safely disable the rule
     'react/react-in-jsx-scope': 'off',
+    // Nop, imagine utils.ts or shared-functions.ts having to export a default function, doesn't make sense
```

Comment:
Does this cause the build error:
`Error:   1:15  error  No named exports found in module './SelectFolder'  import/export`

Reply:
Yeah, it is forcing every file to have a default export. This makes sense for React components, so you write it like
```ts
export default function MyComponent() {}
```
and import it as
```ts
import MyComponent from 'src/components/MyComponent'
// as opposed to import { MyComponent } ...
```
But, it doesn't make sense for other files like utils.ts where there is no single default function or object to export.